test/scylla-cli/run

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash -e
```

Comment:
Although I tend to accept this patch as-is, an even better approach would have been to not create yet another copy of all this logic of running Scylla in a temporary director. After @kostja and others asked me to, I move this logic that was the same in Alternator and CQL-pytest tests to a common Python library, test/cql-pytest/run.py, and now test/alternator/run is a much simpler piece of code (take a look) which doesn't repeat the logic of how to run a temporary Scylla.